app/controllers/posts_controller.rb

```diff
@@ -12,7 +12,7 @@ def show
 
```

Comment:
- [ ] On top of your file use CanCanCan for Post authorization
 
```
  class PostsController < ApplicationController
    load_and_authorize_resource

```

Reply:
Hi, @GzimAsani Thanks for the review 💯
I forgot to do it. Thanks for mentioning it 👍